content/billing/scout-billing.md

```diff
@@ -6,92 +6,94 @@ keywords: payments, billing, subscription, scout
+
+Here you can find:
 
-Renewals charge to the original credit card used to buy Docker Scout Team. You can manage your payment methods in the billing details portal.
+- The type of billing cycle you have, annual or monthly
+- How many repositories are Scout-enabled
+- Your subscription renewal date
+
+Renewals charge to the original credit card used to buy Docker Scout Team. To update your payment method, see [Add or update a payment method](/billing/payment-method/).
 
 ### Update number of repositories for your subscription
 
-You can add or remove the number of repositories associated with your subscription.
+On a Docker Scout Team plan, you can add or remove the number of repositories associated with your subscription. The number of repositories you can add or remove are bundled in groups of five.
 
 To add repositories:
 
-1. In Docker Hub, select **Organizations**.
-2. Select the organization namespace.
-3. Select the **Billing** tab to find your active Docker Scout plan.
-4. Select **Add repositories**.
-5. Enter the number of repositories you want to add, then select **Purchase**.
+1. Sign in to [Docker Scout](https://scout.docker.com/) with your Docker ID.
+2. Select your personal account or organization namespace from the drop-down in the navigation menu on the left side of the screen.
+3. Select **Settings**.
+4. Select **Billing**.
+5. Select **Add repositories**.
+6. Select the number of repositories you want to add, then select **Purchase**.
 
 This purchase charges your default payment method.
 
 To remove repositories:
 
-1. In Docker Hub, select **Organizations**.
-2. Select the organization namespace.
-3. Select the **Billing** tab to find your active Docker Scout plan.
-4. Select **Remove repositories**.
-5. Enter the number of repositories you want to remove, then select **Remove**.
+1. Sign in to [Docker Scout](https://scout.docker.com/) with your Docker ID.
+2. Select your personal account or organization namespace from the drop-down in the navigation menu on the left side of the screen.
+3. Select **Settings**.
+4. Select **Billing**.
+5. Select **Remove repositories**.
+6. Select the number of repositories you want to remove, then select **Remove**.
 
 The number of repositories updates on your next billing cycle.
 
 > **Note**
 >
 > You can't add repositories while repository removal is pending. You can add repositories again during the next billing cycle.
+>
 
 ### Downgrade your subscription
 
-You can downgrade your Docker Scout Team subscription at anytime before the renewal date. Your subscription features remain available until the end of your billing cycle.
+You can downgrade from Docker Scout Team or Docker Scout Business to a Docker Scout Free subscription at anytime before the renewal date. Your subscription features remain available until the end of your billing cycle.
+
+> **Note**
+>
+> You can't downgrade from Docker Scout Business to Docker Scout Team.
```

Comment:
This feels like a bit of a dead end (and strictly speaking isn't true; you first need to cancel Business and then purchase Team). So I might phrase this as "To downgrade from Docker Scout Business to Docker Scout Team, you first need to cancel your Docker Scout Business subscription by contacting your Account Executive". I think the contact action can put the user into a Support ticket.

Reply:
Got it, will update. Is there a way to access a support form from the Scout dashboard? Or do you mean this form in Hub: https://hub.docker.com/support/contact

Reply:
Note: linked to the main support form above.